lib/client/signalflow/signalflow_client.js

```diff
@@ -84,7 +130,8 @@ function SignalflowClient(apiToken, options) {
     },
     preflight: function (opts) {
       return signalflowRequest(opts, 'preflight');
-    }
+    },
+    liveTail: liveTailRequest,
```

Comment:
Nit: all lowercase `livetail` to match the casing of other request types.

Reply:
done